app/models/form_profiles/va_0873.rb

```diff
@@ -36,24 +36,23 @@ def prefill
     super
   end
 
+  private
+
+  # Initializes the personal information for the form with proper error handling
   def initialize_personal_information
-    service_number = profile.is_a?(Hash) ? profile : profile.service_number
+    service_number = profile.is_a?(Hash) ? profile[:service_number] : profile.service_number
 
+    # Merge the user's normalized name, preferred name, and service number into the payload
     payload = user.full_name_normalized.merge(preferred_name:, service_number:)
 
     VA0873::FormPersonalInformation.new(payload)
   rescue => e
-    log_exception_to_sentry(e, {}, prefill: :personal_information)
-    {}
+    handle_exception(e, :personal_information)
```

Comment:
Same question about PII

Reply:
None, it'll just log the error. No PII will be in the error message.